tasks/CMakeLists.txt

```diff
@@ -8,7 +8,9 @@ add_dependencies(${ROCK_GAZEBO_TASKLIB_NAME}
 
 TARGET_LINK_LIBRARIES(${ROCK_GAZEBO_TASKLIB_NAME}
     ${OrocosRTT_LIBRARIES}
-    ${ROCK_GAZEBO_TASKLIB_DEPENDENT_LIBRARIES})
+    ${ROCK_GAZEBO_TASKLIB_DEPENDENT_LIBRARIES}
+    thruster_types)
```

Comment:
I've actually created thruster_types inside **simulation/gazebo_thruster**. [Look inside src/] (https://github.com/Brazilian-Institute-of-Robotics/simulation-gazebo_thruster) 
Maybe I made something wrong with its cmakefiles because the rock component doesn't find the package when I use: 

    using_library 'name_of_pkgconfig_package'